adam_preprocessing/shape_stroke_extraction.py

```diff
@@ -502,6 +527,15 @@ def get_strokes(self):
                         strokes_normalized_coordinates=reduced_strokes_norm.transpose(
                             0, 2, 1
                         ).tolist(),
+                        size=dict(
```

Comment:
I think it probably makes things easier to put this at the outer level -- so at the same level as `relative_distance` instead of being nested inside `stroke_graph`.